src/package.xml

```diff
@@ -52,6 +52,10 @@
         <members>CON_Email_TEST</members>
         <members>CON_EthnicityRace_BATCH</members>
         <members>CON_EthnicityRace_TEST</members>
+        <members>CON_Phone</members>
+        <members>CON_Phone_Batch</members>
+        <members>CON_PreferredPhone_TDTM</members>
+        <members>CON_PreferredPhone_TEST</members>
```

Comment:
@skosgi1  I'm getting a deployment error when trying to deploy the latest changes to my org. I think it's because this was renamed to CON_PreferredPhone_TEST in the package.xml, but the actual file wasn't changed, perhaps?

`[Failed]: Update of ApexClass CON_PreferredPhone_Test: Error: Not in package.xml`